### DISCUSSION TITLE
missing pop-out toolbar when double monitor

### DISCUSSION THREAD

**Discussion by andrearicci** [pubpub-zz/ppInk#257] [General]
when I'm working with two monitors ("extend" mode), I miss the pop-out toolbar
this is all I see: 
![image](https://user-images.githubusercontent.com/1193266/233079464-d02c5756-a22b-4688-afc1-deb1eee5b373.png)
Clicking on it I only obtain the default pen.
Thank you

**Comment by pubpub-zz** [comment]
This can occur due your layout. To restore it you should:
open the options( right click on the ppink icon  in the notification area)
uncheck "allow dragging toolbar" to force its position on the edge of the primary screen
![image](https://user-images.githubusercontent.com/4083478/233419731-40b311e9-2e8f-439b-89ab-220bf8d83f0e.png)
Do not close the dialog box,
Click on the floating icon, the toolbar should occur.
You can then close the toolbar reactivate the dragging option in the options, reopen the toolbar to move it where you want, close it again then "save to files" your configuration. you can then close the options

**Comment by andrearicci** [comment]
Thank you but I had no luck. 
Actually when I'm in double monitor configuration, if I open the options, and start annotating, the cursor is about 1/3 of the screen far away from the cursor.
Maybe for this reason I'm not able to move the toolbar nor change the tool: the "click" coordinates are not the same of the cursor.

![image](https://user-images.githubusercontent.com/1193266/233590253-719d018b-a088-4441-b792-f0a7a076e77c.png)
Thank you

**Comment by pubpub-zz** [comment]
have a look there:
https://github.com/pubpub-zz/ppInk#tips

**Comment by andrearicci** [comment]
yes!, that's it! thank you

**Comment by pubpub-zz** [comment]
👍 do not hesitate to star it if you like it.
